projects/extension/sql/idempotent/001-openai.sql

```diff
@@ -56,17 +56,17 @@ set search_path to pg_catalog, pg_temp
 -- generate an embedding from a text value
 -- https://platform.openai.com/docs/api-reference/embeddings/create
 create or replace function ai.openai_embed
-( _model text
-, _input text
-, _api_key text default null
-, _base_url text default null
-, _dimensions int default null
-, _user text default null
+( model text
+, "input" text
```

Comment:
```suggestion
, input_text text
```

Reply:
The postgres docs say input is not actually reserved so we can use it just fine as far as I understand

Reply:
it's reserved under SQL-92

Reply:
Fair, but does that matter for our use case? I mean that standard is 36 years old and since postgres has relaxed the constraints I don't see why we would adhere to these constraints.
It's unlikely we would port this over to another database system isn't it?

Reply:
🤷‍♂️ this is why i've always used an underscore prefix